charts/kyverno/templates/admission-controller/deployment.yaml

```diff
@@ -120,7 +120,7 @@ spec:
           image: {{ include "kyverno.image" (dict "image" .Values.admissionController.container.image "defaultTag" .Chart.AppVersion) | quote }}
           imagePullPolicy: {{ .Values.admissionController.container.image.pullPolicy }}
           args:
-            - --backgroundServiceAccountName='system:serviceaccount:{{ include "kyverno.namespace" . }}:{{ include "kyverno.background-controller.serviceAccountName" . }}'
+            - --backgroundServiceAccountName=system:serviceaccount:{{ include "kyverno.namespace" . }}:{{ include "kyverno.background-controller.serviceAccountName" . }}
```

Comment:
It took me a whole day to find this quote issue!

Reply:
The single quote is literally treated as part of the service account name string, so it fails access checks using SAR. Having a single quote is something like `kubectl auth ... --as 'system:serviceaccount:xx:xx'`, of course if fails the check.

Reply:
Good catch !